Resources/config/services.xml

```diff
@@ -4,134 +4,183 @@
     <services>
         <!-- The league interfaces -->
         <service id="League\OAuth2\Server\AuthorizationServer" alias="league.oauth2.server.authorization_server" />
+        <service id="league.oauth2.server.authorization_server" alias="League\OAuth2\Server\AuthorizationServer" />
```

Comment:
@rjwebdev This seems to be a duplicate, you already have this on line 78.

Reply:
Should be removed.